sphinx/source/performance.rst

```diff
@@ -0,0 +1,102 @@
+Performance
+===========
+
+Overview
+--------
+
+CCF pairs strong confidentiality guarantees with extremely high performance. CCF can sustain high transaction throughputs while achieving global commits with low latency.
```

Comment:
"extremely high" -> very high?
"throughputs" -> throughput?
"global commits" -> This is not a term that we use much in the docs. Should we say "appending transactions to the ledger" instead?

Reply:
If we are mentioning high performance/throughput etc we should add latest numbers or at least a link to the TR where the latest numbers are mentioned.

Reply:
Changed the wording here, and added link to the TR.